cli/resource.go

```diff
@@ -72,3 +77,28 @@ func runResourceAdd(args *docopt.Args, client *controller.Client) error {
 
 	return nil
 }
+
+func runResourceRemove(args *docopt.Args, client *controller.Client) error {
+	provider := args.String["<provider>"]
+	resource := args.String["<resource>"]
+
+	res, err := client.DeleteResource(provider, resource)
+	if err != nil {
+		return err
+	}
+
+	// Unset all the keys associated with this resource
+	env := make(map[string]*string)
+	for k := range res.Env {
+		env[k] = nil
```

Comment:
What do you think about comparing the values too? It's totally possible that they were changed since the resource was added. (long-term we need to come up with a more robust way of handling resource config)